src/helpers/bookings.js

```diff
@@ -0,0 +1,17 @@
+import models from '../db/models';
+
+const { Bookings } = models;
+class bookingHelper {
+  static async searchBooking(id) {
+    const getBookings = await Bookings.findAll({
```

Comment:
@Niyonsengaeric please don't include queries in this file, find a file called queries and move this queries there

Reply:
let me remove it